config/config.go

```diff
@@ -1015,3 +1022,27 @@ func (re Regexp) MarshalYAML() (interface{}, error) {
 	}
 	return nil, nil
 }
+
+// RemoteWriteConfig is the configuration for remote storage.
+type RemoteWriteConfig struct {
+	Hostname      string         `yaml:"hostname,omitempty"`
```

Comment:
This needs to be an address (hostname:port), not just a hostname. So `Address` would be the Go-ish terminology for that. I guess that also makes sense to call it that in the config file.